app/frontend/src/features/auth/signup/AccountForm.tsx

```diff
@@ -87,238 +89,259 @@ export default function AccountForm() {
   const { authState, authActions } = useAuthContext();
   const authLoading = authState.loading;
 
-  const { control, register, handleSubmit, errors } =
+  const { control, register, handleSubmit, errors, watch } =
     useForm<SignupAccountInputs>({
       defaultValues: { location: { address: "" } },
       mode: "onBlur",
       shouldUnregister: false,
     });
 
-  const [loading, setLoading] = useState(false);
-  const [acceptedTOS, setAcceptedTOS] = useState(false);
-
   const classes = useStyles();
   const authClasses = useAuthStyles();
 
-  const submit = handleSubmit(async (data: SignupAccountInputs) => {
-    authActions.clearError();
-
-    if ((data.location?.address ?? "") === "") {
-      authActions.authError(SIGN_UP_LOCATION_MISSING);
-      return;
+  const mutation = useMutation<void, GrpcError, SignupAccountInputs>(
+    async ({
+      username,
+      birthdate,
+      gender,
+      acceptTOS,
+      hostingStatus,
+      location,
+    }) => {
+      const state = await service.auth.signupFlowAccount(
+        {
+          flowToken: authState.flowState?.flowToken!,
+          username: lowercaseAndTrimField(username),
+          birthdate: birthdate.format().split("T")[0],
+          gender,
+          acceptTOS,
+          hostingStatus,
+          city: location.address,
+          lat: location.lat,
+          lng: location.lng,
+          radius: location.radius,
+        }
+        // TODO: password
```

Comment:
Is this a backend limitation at the moment with the sign up RPC (that you can't supply password during sign up)?

Reply:
No we just didn't put it in frontend yet